src/artifacts_keyring/plugin.py

```diff
@@ -22,6 +22,8 @@ class CredentialProvider(object):
 
     def __init__(self):
         if sys.platform.startswith("win"):
+            # by default, attempt to search netfx plugins folder.
+            # if that doesn't exist, search netcore for newer credprovider versions.
```

Comment:
We control what versions are available in the folder during the setup.py. Should we just switch over to dotnet for both? I dont see any benefits of netfx and netcore.

Reply:
I'd like to plan for a 1.0 release which deprecates the NET Framework bits fully, but until then we should continue to support both.